<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWin</class>
 <widget class="QMainWindow" name="MWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>555</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>555</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>华科寝室用电查询系统 v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/logo</normaloff>:/images/logo</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>555</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="query">
         <attribute name="title">
          <string>电费查询</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="subLayout" stretch="0,0,1">
            <item>
             <widget class="QLabel" name="timeLabel">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>欢迎登陆本系统，今天是2017/8/28 15:12</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="hintLabel">
              <property name="text">
               <string>操作：请依次选择区域、楼栋、房间，点查询完成查询操作！</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>( ・◇・)？</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="layout1" stretch="1,3,1,3">
                 <item>
                  <widget class="QLabel" name="areaLabel">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>区域：</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="areaBox">
                   <item>
                    <property name="text">
                     <string>-请选择-</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>东区</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>留学生楼</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>西区</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>韵苑</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>紫菘</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="buildingLabel">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>楼号：</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="buildingBox"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="layout2" stretch="1,3,2,2,0">
                 <item>
                  <widget class="QLabel" name="roomLabel">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>房间：</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="roomEdit"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="queryBtn">
                   <property name="text">
                    <string>查询</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="cancelBtn">
                   <property name="text">
                    <string>取消</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="help">
                   <property name="maximumSize">
                    <size>
                     <width>0</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string> </string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="layout3" stretch="1,3,1,3">
                 <item>
                  <widget class="QLabel" name="checkLabel">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>抄表时间：</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="checkEdit">
                   <property name="readOnly">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="leftLabel">
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>剩余电量：</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="leftEdit">
                   <property name="readOnly">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="layout4">
                 <item>
                  <widget class="QTextEdit" name="chargeTable">
                   <property name="verticalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                   </property>
                   <property name="horizontalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                   </property>
                   <property name="readOnly">
                    <bool>false</bool>
                   </property>
                   <property name="placeholderText">
                    <string>Ctrl+h 获取帮助</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="info">
         <attribute name="title">
          <string>关于</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QGroupBox" name="bugReportBox">
            <property name="title">
             <string>bug反馈</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="layout6" stretch="1,6">
               <item>
                <widget class="QLabel" name="githubLabel">
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>GitHub: </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="githubEdit">
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>https://github.com/LewisTian/PyQt5-Apps</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="layout7" stretch="1,6">
               <item>
                <widget class="QLabel" name="emailLabel">
                 <property name="text">
                  <string>Email: </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="emailEdit">
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>lewissmith@126.com</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="offlineBox">
            <property name="title">
             <string>上班时间：每日17点至22点（节假日另见充值点通知）</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="infoLabel">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>      紫菘二栋充值点： 每周一、二、四、五、日    
      西八舍充值点：    每周一、二、三、五、六   
      东八舍充值点：    每周一、三、四、五、六    
      韵苑五栋充值点： 每周一、三、四、五、六
      韵苑21栋充值点：每周一、二、四、五、日</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="layout5">
            <item>
             <widget class="QLabel" name="onlineChargeLabel">
              <property name="text">
               <string>在线充值：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="onlineChargeEdit">
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>http://sdhq.hust.edu.cn/ICBS/mobileweb/html/index.html</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="layout8">
            <item>
             <widget class="QLabel" name="onlineQueryLabel">
              <property name="text">
               <string>在线查询：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="onlineQueryEdit">
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>http://202.114.18.218/Main.aspx</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{  
   color: rgba(0, 0, 0, 50%);
}</string>
            </property>
            <property name="text">
             <string>—— 我是有底线的 ——</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
